File: src/1-Two-Sum-Easy.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
	        for (int i=0;i<nums.length;i++){
	            for (int j=0;j<nums.length;j++){
	                if(i==j)
	                    continue;
	                else if ((nums[i]+nums[j])==target){
	                    int result[] = new int[2];
	                    result[0]=i;
	                    result[1]=j;
	                    return result;
	                }
	            }
	        }
	        return null;
	    }
}

File: src/1-Two-Sum-Easy.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> r;
        for(int i=0;i<nums.size()-1;i++){
          for(int j=i+1;j<nums.size();j++){
                if(nums.at(i)+nums.at(j)==target){
                    r.push_back(i);
                    r.push_back(j);
                }     
            }
        }
        return r;
    }
};
